src/ralph/data_importer/resources.py

```diff
@@ -182,6 +183,11 @@ class Meta:
         model = physical.Rack
 
 
+class DiscoveryDataCenterResource(RalphModelResource):
```

Comment:
please add some comment that "this is another resource for `DataCenter` to handle another source of data from Ralph2 (discovery DataCenter)"